src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailykuudra/DailyKuudraBossHelper.kt

```diff
@@ -75,7 +78,7 @@ class DailyKuudraBossHelper(private val reputationHelper: CrimsonIsleReputationH
                 lineList.add(" ")
                 lineList.add(displayItem.getItemStack())
                 lineList.add("$displayName: $result")
-                display.add(lineList)
+                if (!(config.hideComplete.get() && tier.doneToday)) display.add(lineList)
```

Comment:
You can move the "if" check to the first line in the for loop (early continue so that the whole lineList logic doesn't happen)